### DISCUSSION TITLE
Sliding Window Inference explanation

### DISCUSSION THREAD

**Discussion by emi-dm** [Project-MONAI/MONAI#7853] [Q&A] [answered]
Hi, can anyone tell me if there is a tutorial explaining how the Sliding Window Inference method works?
References on how it works would also be appreciated.

Thanks!!

**Comment by KumoLiu** [comment]
Hi @emi-dm, here is a tutorial showing how sliding_window_inference works.
https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb

Hope it helps, thanks.

**Reply by emi-dm** [reply]
Thanks @KumoLiu!! But I meant more in detail, i.e. how does it take the windows, how does it overlap, what does that overlap influence. could you provide me with information about it? Thank you very much for taking the time to answer!

**Reply by KumoLiu** [reply, marked as answer]
Perhaps you can refer to this tutorial: https://github.com/Project-MONAI/tutorials/blob/main/patch_inferer/modular_patch_inferer.ipynb

**Reply by emi-dm** [reply]
Thanks @KumoLiu!!!
